arp-speaker/main.go

```diff
@@ -201,7 +201,7 @@ func main() {
 	if err != nil {
 		glog.Fatalf("Error setting up leader election: %s", err)
 	}
-	go func() { le.Run() }()
+	le.Run()
```

Comment:
This looks like it changes the semantics to block before client.Run(), so the controller never starts executing. Are you sure?